.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: false
 language: java
 
 jdk:
-   - openjdk8
+   - openjdk11
```

Comment:
Btw there is manual JDK 11 installation in before_script && script, maybe we can also remove it

Reply:
I think we must keep openjdk8 there as well. I plan to switch this to matrix build. We need to build both JDK11 and JDK8, at least until we can drop JDK8 support.